packages/pages/package.json

```diff
@@ -50,7 +50,8 @@
     "generate-notices": "generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES --overwrite",
     "prepublishOnly": "pnpm run build:release",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "prepare": "pnpm run build:release"
```

Comment:
we should rename the build step to `build:prepare` if we want to run it on the prepare hook.

Reply:
It's also used in `prepublishOnly` above. You still want to rename to `build:prepare`?